scripts/capa2yara.py

```diff
@@ -0,0 +1,751 @@
+"""
+Convert capa rules to YARA rules (where this is possible)
+
+    usage: capa2yara.py [-h] [--private] [--version] [-v] [-vv] [-d] [-q] [--color {auto,always,never}] [-t TAG] rules
+
+Capa to YARA rule converter
+
+positional arguments:
+  rules                 Path to rules
+
+optional arguments:
+  -h, --help            show this help message and exit
+  --private, -p         Create private rules
+  --version             show program's version number and exit
+  -v, --verbose         enable verbose result document (no effect with --json)
+  -vv, --vverbose       enable very verbose result document (no effect with --json)
+  -d, --debug           enable debugging output on STDERR
+  -q, --quiet           disable all output but errors
+  --color {auto,always,never}
+                        enable ANSI color codes in results, default: only during interactive session
+  -t TAG, --tag TAG     filter on rule meta field values
+
+
+Copyright (C) 2020, 2021 Arnim Rupp (@ruppde) and FireEye, Inc. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+You may obtain a copy of the License at: [package root]/LICENSE.txt
+Unless required by applicable law or agreed to in writing, software distributed under the License
+ is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and limitations under the License.
+"""
+
+import re
+import pdb
+import sys
+import string
+import logging
+import argparse
+import datetime
+import itertools
+
+import capa.main
+import capa.rules
+import capa.engine
+import capa.features
+import capa.features.insn
+from capa.features import ARCH_X32, ARCH_X64, String
+from capa.features.insn import Number, Offset
+
+logger = logging.getLogger("capa2yara")
+
+today = str(datetime.date.today())
+
+# create unique variable names for each rule in case somebody wants to move/copy stuff around later
+var_names = ["".join(letters) for letters in itertools.product(string.ascii_lowercase, repeat=3)]
```

Comment:
alternatively, rather than realizing the generator into a concrete list, could use just the generator, and call `next(var_names)` when fetching the next term.

upside is that there is less upfront computation and memory overhead.

this is just an FYI, no need to change here.